README.md

```diff
@@ -206,6 +223,11 @@ both to download, but also to run the monthly analysis is not currently tested.
 This tool downloads the data, but also includes several functions which create symbolic links
 in the data's directory in order to accomodate incompatible changes in NEMO's output formatting.
 
+Please consult the command help for more details:
+```
+download_from_mass -h
```

Comment:
can you add this to the commands we test in `.github/workflows/run-tests.yml` as well pls?

Reply:
did it now